src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,21 @@
+package racingcar;
+
+public class Car {
+
+    public static final int MOVE_LIMIT = 4;
+    private int distance = 1;
+
+    public void move(int random) {
```

Comment:
```suggestion
    public void move(int 거리 ? value ?) {
```

`Car` 입장에서 `random`이라는 개념을 알고있을 필요가 있을지 고민해보아요!

Reply:
생각해보니 Car에서는 알 필요가 없네요! 반영하겠습니다.